site/docs/main/configuration.md

```diff
@@ -186,6 +187,31 @@ If present, the value of the `CONTOUR_NAMESPACE` environment variable is used as
 
 The `CONTOUR_NAMESPACE` environment variable is set via the [Downward API][6] in the Contour [example manifests][7].
 
+## Bootstrap Config File
+
+The boostrap configuration file is generated by an initContainer in the Envoy daemonset which configures the Envoy
+container to connect to Contour and receive configuration via xDS.
+
+### Arguments
+
+There are arguments that can be passed to `contour boostrap` that help configure how Envoy
+connects to Contour:
+
+| Arg | Default  | Description |
+|------------|----------|-------------|
+| <nobr>resources-dir</nobr> | "" | Defines the directory where resource files will be written.  |
+| <nobr>admin-address</nobr> | 127.0.0.1 | Defines the address the Envoy Admin Webpage will listen on.  |
+| <nobr>admin-port</nobr> | 9001 | Defines the port the Envoy Admin Webpage will listen on.  |
+| <nobr>xds-address</nobr> | 127.0.0.1 | Defines the address to connect to Contour xDS server on.  |
+| <nobr>xds-port</nobr> | 8001 | Defines the port to connect to Contour xDS server on. |
+| <nobr>envoy-cafile</nobr> | "" | Defines the gRPC CA Filename for Envoy to load.  |
+| <nobr>envoy-cert-file</nobr> | "" | Defines the gRPC Client cert filename for Envoy to load.  |
+| <nobr>envoy-key-file</nobr> | "" | Defines the gRPC Client key filename for Envoy to load.  |
+| <nobr>namespace</nobr> | projectcontour | Defines the namespace the Envoy container will run in. Also configured via ENV variable "CONTOUR_NAMESPACE".   |
```

Comment:
```suggestion
| <nobr>namespace</nobr> | projectcontour | Defines the namespace the Envoy container will run in. If this flag is not given, the value of the "CONTOUR_NAMESPACE" environment variable is used.   |
```

You should describe what the flag actually does (i.e. why someone might need to set it).